pandas/tools/pivot.py

```diff
@@ -382,7 +382,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     columns : array-like, Series, or list of arrays/Series
         Values to group by in the columns
     values : array-like, optional
-        Array of values to aggregate according to the factors
+        Array of values to aggregate according to the factors.
```

Comment:
but you are adding a default, that was not discussed. nor do I see a good reason for it. pls justify.